sinon/lib/util/Wrapper.py

```diff
@@ -2,6 +2,11 @@
 Copyright (c) 2016-2017, Kir Chou
 https://github.com/note35/sinon/blob/master/LICENSE
 """
+import traceback
+import sys
+sys.path.insert(0, '../../')
+from lib.call import Call
```

Comment:
This will cause CI failed, fixing the folder structure might resolve this at the same time.